mlir/test/Dialect/Vector/vector-transfer-flatten.mlir

```diff
@@ -551,6 +620,8 @@ func.func @negative_out_of_bound_transfer_read(
 
 // -----
 
+// Can flatten the righmost dynamic dimension
```

Comment:
"Can"? But it's not flattened?

Reply:
That comment is accidental, no idea where is came from.